lib/hyperswitch.js

```diff
@@ -295,10 +295,7 @@ HyperSwitch.prototype.defaultListingHandler = function (match, hyper, req) {
             match.value.specRoot && !match.value.specRoot['x-listing']) {
         return P.resolve({
             status: 200,
-            body: Object.assign({}, match.value.specRoot, {
-                // Set the base path dynamically
-                basePath: getDocBasePath(req, match.value.specRoot)
```

Comment:
This was done for a reason - we used `x-host-basePath` stanza in the restbase config in order to set it to `/api/rest_v1` that's used in production. I don't think this will work anymore. You would probably need to replace the `servers` stanza

Reply:
Are you sure? Just ran it locally and `x-host-basePath` is still set to `/api/rest_v1`

Reply:
`x-host-basePath` is set, yes, but I don't think it's translated to `servers`

Reply:
I don't understand. servers[0].url is set to `/en.wikipedia.org/v1`. Are you saying `/api/rest_v1` should also be added there?

Reply:
Basically, we have 2 modes on how we expose RESTBase.

Locally, you get the API under `localhost:7231/{domain}/v1/` - for this the path is correct. However, in production RESTBase is exposed under `{domain}/api/rest_v1`, - so in production we need to set the basePath (which in swagger 3 is servers[0].url AFAIK) to `/api/rest_v1`, which is what getDocBasePath function will return.

Reply:
Done.